test/integration/JsonSerialization.test.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  BundleEntry,
+  Coding,
+  Extension,
+  KitchenSink,
+  PrimitiveBoolean,
+  PrimitiveCode,
+  PrimitiveString,
+  PrimitiveTime,
+  PrimitiveUri,
+  ResourceChild,
+} from "../fixture/generatedTypeScript/fhir";
+
+describe("Testing the JSON serialization logic of a generated class", () => {
+  it("should serialize a simple object", () => {
+    const input = new Coding();
+    input.version = new PrimitiveString();
+    input.version.value = "versionX";
+    input.system = new PrimitiveUri(); // no value set
+    input.code = undefined; // explicitly set to undefined
+    // display and userSelected are undefined implicitly
+
+    const result = JSON.stringify(input, null, 2);
+    expect(result).toBe(`{
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)